Makefile

```diff
@@ -200,6 +204,14 @@ install:
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 ./tetragon $(DESTDIR)$(BINDIR)
 
+.PHONY: install-operator 
+install-operator:
+	./contrib/localdev/install-tetragon.sh --image cilium/tetragon:latest --operator cilium/tetragon-operator:latest
+
+.PHONY: setup
+setup-kind: build kind install-operator 
```

Comment:
The rule name doesn't match `.PHONY` here, but I don't think this rule is necessary - if we add dependencies to the one above, we can just use that.